nixos/modules/installer/tools/nixos-rebuild.sh

```diff
@@ -111,17 +115,17 @@ buildHostCmd() {
     if [ -z "$buildHost" ]; then
         "$@"
     elif [ -n "$remoteNix" ]; then
-        ssh $SSHOPTS "$buildHost" env PATH="$remoteNix:$PATH" "$@"
+        ssh $SSHOPTS "$buildHost" env PATH="$remoteNix:$PATH" "$maybeSudo$@"
```

Comment:
Security policies might prevent `sudo` from passing `PATH`, so maybe something like @edolstra [suggested](https://github.com/NixOS/nixpkgs/pull/70734#issuecomment-541027201) might be needed after all.

Reply:
Good point. At least this works well with NixOS, I tested with my setup:

```
  security.sudo = {
    enable = true;
    wheelNeedsPassword = false;
  };
```

(I don't want to do change how PATH is handled here, as I feel it's orthogonal to this PR.)